### PR TITLE
Add Error codes for Server Errors from 520 to 526 (core/aws-messages)

### PR DESCRIPTION
**Pull request by ax-hack**
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#Cloudflare

### CODE REVIEW

File: src/core/aws-messages.adb [hunk 1]
```diff
@@ -103,6 +103,13 @@ package body AWS.Messages is
    S508_Message : aliased constant String := "Loop Detected";
    S510_Message : aliased constant String := "Not Extended";
    S511_Message : aliased constant String := "Network Authentication Required";
+   S520_Message : aliased constant String := "Unknown Error";
+   S521_Message : aliased constant String := "Web Server Is Down";
+   S522_Message : aliased constant String := "Connection Timed Out";
+   S523_Message : aliased constant String := "Origin Is Unreachable ";
```
Comment (TurboGit) [inline src/core/aws-messages.adb:109]: Ending space before " should be removed.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Minor change needed, apart from that it looks fine. Thanks.

File: src/core/aws-messages.adb (hunk 1 above)
Comment (ax-hack) [reply src/core/aws-messages.adb]: Fixed!

**Review by TurboGit — APPROVED** [review]
Looks good now, thanks!

**Comment by TurboGit** [comment]
I'll merge in the coming days.

**Pull request merged**